packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -98,10 +128,6 @@ export class AccordionItem implements ConditionalSlotComponent {
 
   connectedCallback(): void {
     this.parent = this.el.parentElement as HTMLCalciteAccordionElement;
```

Comment:
The parent could probably be set as well.

Reply:
Hmm... not sure how to achieve this. What do you have in mind?

Reply:
What I was thinking is that the owning accordion could set itself on the items as the parent. Just like how its setting the icontype, iconPosition, scale.